IPython/terminal/interactiveshell.py

```diff
@@ -286,6 +286,16 @@ def _make_style_from_name_or_cls(self, name_or_cls):
                     Token.OutPrompt: '#990000',
                     Token.OutPromptNum: '#ff0000 bold',
                 })
+
+                # Hack: Due to limited color support on the Windows console
+                # the prompt colors will be wrong without this
+                if os.name == 'nt':
+                    style_overrides.update({
+                        Token.Prompt: '#ansidarkgreen',
```

Comment:
It's actually prompt-toolkit that handles this. They are declared in [prompt_toolkit/styles/base.py:39-48](https://github.com/jonathanslenders/python-prompt-toolkit/blob/99fa7fae61c9b4ed9767ead3b4f9b1318cfa875d/prompt_toolkit/styles/base.py#L39-L48) and referenced throughout prompt-toolkit.

I tried using hex colors but prompt-toolkit seems quite stubborn to use the dark versions instead of the light versions...